### PR TITLE
PWN-4512 - Fix Token List Diff Utils

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket
https://p2pvalidator.atlassian.net/browse/PWN-4512
https://p2pvalidator.atlassian.net/browse/PWN-4483

## Description of Work

Fix bug with Token List Header, and change size of Bottom Nav Bar

### CODE REVIEW

**Review by kuznetsov — APPROVED** [review]

File: app/src/main/res/layout/item_home_token_list_item_shimmer.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingHorizontal="16dp"
+    android:paddingVertical="8dp">
+
+    <ImageView
+        android:id="@+id/iconView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_home_token_list_item_shimmer.xml:11]: imageViewTokenIcon

File: app/src/main/res/layout/item_home_token_list_item_shimmer.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        android:layout_width="44dp"
+        android:layout_height="44dp"
+        android:background="@drawable/bg_stroke_rounded"
+        android:backgroundTint="@color/bg_rain"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <ImageView
+        android:id="@+id/topLeftView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_home_token_list_item_shimmer.xml:21]: imageViewTopLeftSkeleton

File: app/src/main/res/layout/item_home_token_list_item_shimmer.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        app:layout_constraintBottom_toBottomOf="@id/iconView"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginBottom="6dp"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toEndOf="@id/iconView"
+        app:layout_constraintWidth_percent="0.3" />
+
+
+</androidx.constraintlayout.widget.ConstraintLayout>
+
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_home_token_list_item_shimmer.xml:50]:
```suggestion
```

File: app/src/main/res/layout/item_home_token_list_shimmer.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_home_token_list_shimmer.xml:2]:
```suggestion
```

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**